test/utils/image/manifest.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const (
+	e2eImagesRegistry  = "gcr.io/kubernetes-e2e-test-images"
+	BusyBox            = "busybox"
```

Comment:
I think `gcr.io/google-containers/busybox:1.24` is just the official docker `busybox:1.24` that was downloaded at some point and pushed back to `gcr.io/google-containers`, since gcr.io was/is more reliable than dockerhub (https://github.com/kubernetes/kubernetes/issues/13288).

as with many things, I'm not sure if this is still necessary.